geode-core/src/test/java/org/apache/geode/internal/net/BufferPoolTest.java

```diff
@@ -133,27 +135,42 @@ public void checkBufferSizeAfterAllocation() throws Exception {
 
   @Test
   public void checkBufferSizeAfterAcquire() throws Exception {
+    // allocate a small buffer and a larger buffer. Check their sizes, etc and then
+    // release and reacquire them. They should be from separate buffer pools so there
+    // should still be a small buffer and a larger buffer.
```

Comment:
I'm retracting that comment since these variable names existed before this PR.

Any such changes should be pursued in connection with a refactoring ticket.